TTS: state that only catalog voices resolve; document custom/cloned voice options (#535)
signalwire/docs: fern/products/platform/pages/calling/voice/TTS/custom-voices.mdx

A voice ID from your own provider account — including custom or cloned voices — is not visible
to SignalWire's account. It will not resolve, and TTS falls back to a default voice at call time.
This applies even though cloned-voice IDs can appear identical in format to a catalog voice ID.
</Warning>

### ElevenLabs

ElevenLabs custom and cloned voices can be imported into SignalWire's catalog by request.
[Contact Support](https://support.signalwire.com/portal) to request a voice import.
Once the voice is imported, use it with the standard
[voice string format](/docs/platform/voice/tts/elevenlabs#usage):

```
elevenlabs.<voice_id>:<model>
```

### Cartesia

Cartesia custom and cloned voices are not supported.
Use a voice from the catalog on the
[Cartesia reference page](/docs/platform/voice/tts/cartesia#voices).

## Next steps

<CardGroup cols={2}>
  <Card title="ElevenLabs" href="/docs/platform/voice/tts/elevenlabs">
    Browse the ElevenLabs voice catalog and voice string format.
  </Card>
  <Card title="Cartesia" href="/docs/platform/voice/tts/cartesia">
    Browse the Cartesia voice catalog and voice string format.
  </Card>

> </CardGroup>

Author: ```suggestion
```

---
title: Custom and cloned voices
subtitle: Text-to-speech
slug: /voice/tts/custom-voices
description: Learn which custom and cloned TTS voice options are supported on the SignalWire platform.
max-toc-depth: 3
---

SignalWire's TTS integrations run on SignalWire's own accounts with each provider.
When you set a voice string like `elevenlabs.<voice_id>` or `cartesia.<voice_id>:<model>`,
SignalWire resolves the voice ID against its own provider account —
not yours.

<Warning>
Only voices from SignalWire's catalog resolve on the platform.
A voice ID from your own provider account — including custom or cloned voices — is not visible
to SignalWire's account. It will not resolve, and TTS falls back to a default voice at call time.

> This applies even though a cloned-voice ID looks identical in format to a catalog voice ID.

Author: ```suggestion
This applies even though cloned-voice IDs can appear identical in format to a catalog voice ID.
```

</Warning>

## Supported options by provider

| Provider | Custom and cloned voices |
| :------- | :----------------------- |
| [ElevenLabs](/docs/platform/voice/tts/elevenlabs) | Available by request through Support |
| [Cartesia](/docs/platform/voice/tts/cartesia) | Not supported |

> 

Author: ```suggestion
```
